apps/system/js/callscreen_window.js

```diff
@@ -104,7 +104,6 @@
     this.browserContainer.insertBefore(this.browser.element, null);
     this.frame = this.element;
     this.iframe = this.browser.element;
-    this.screenshotOverlay = this.element.querySelector('.screenshot-overlay');
```

Comment:
nope.
if an app has an inline activity, we will screenshot the inline activity and show this screenshot while edge swiping (since it's on top).